db/migrate/20231102114147_create_likes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateLikes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :likes do |t|
+      t.integer :user_id
+      t.integer :post_id
```

Comment:
- [ ] @iankiema Kindly consider removing these columns, they will be automatically generated when running your migrations using ` t.references :user, foreign_key: true,  t.references :post, foreign_key: true` code.